worlds/hk/ExtractedData.py

```diff
@@ -3,14 +3,16 @@
 starts = {"king's_pass": 'Tutorial_01'}
+vanilla_location_costs = {'Dash_Slash': {'GEO': 800}, 'Unbreakable_Heart': {'GEO': 600}, 'Unbreakable_Greed': {'GEO': 450}, 'Unbreakable_Strength': {'GEO': 750}, 'Vessel_Fragment-Basin': {'GEO': 3000}, 'Crossroads_Map': {'GEO': 30}, 'Greenpath_Map': {'GEO': 60}, 'Fog_Canyon_Map': {'GEO': 150}, 'Fungal_Wastes_Map': {'GEO': 75}, 'Deepnest_Map-Upper': {'GEO': 38}, 'Deepnest_Map-Right': {'GEO': 38}, 'Ancient_Basin_Map': {'GEO': 112}, "Kingdom's_Edge_Map": {'GEO': 112}, 'City_of_Tears_Map': {'GEO': 90}, 'Royal_Waterways_Map': {'GEO': 75}, 'Howling_Cliffs_Map': {'GEO': 75}, 'Crystal_Peak_Map': {'GEO': 112}, "Queen's_Gardens_Map": {'GEO': 150}, 'Resting_Grounds_Map': {'GEO': 56}, 'Crossroads_Stag': {'GEO': 50}, 'Greenpath_Stag': {'GEO': 140}, "Queen's_Station_Stag": {'GEO': 120}, "Queen's_Gardens_Stag": {'GEO': 200}, 'City_Storerooms_Stag': {'GEO': 200}, "King's_Station_Stag": {'GEO': 300}, 'Distant_Village_Stag': {'GEO': 250}, 'Hidden_Station_Stag': {'GEO': 300}, 'Stag_Nest_Stag': {'GEO': 300}, 'Elevator_Pass': {'GEO': 150}}
+vanilla_shop_costs = {('Sly', 'Simple_Key'): [{'GEO': 950}], ('Sly', 'Rancid_Egg'): [{'GEO': 60}], ('Sly', 'Lumafly_Lantern'): [{'GEO': 1800}], ('Sly', 'Gathering_Swarm'): [{'GEO': 300}], ('Sly', 'Stalwart_Shell'): [{'GEO': 200}], ('Sly', 'Mask_Shard'): [{'GEO': 500}, {'GEO': 150}], ('Sly', 'Vessel_Fragment'): [{'GEO': 550}], ('Sly_(Key)', 'Heavy_Blow'): [{'GEO': 350}], ('Sly_(Key)', 'Elegant_Key'): [{'GEO': 800}], ('Sly_(Key)', 'Mask_Shard'): [{'GEO': 1500}, {'GEO': 800}], ('Sly_(Key)', 'Vessel_Fragment'): [{'GEO': 900}], ('Sly_(Key)', 'Sprintmaster'): [{'GEO': 400}], ('Iselda', 'Wayward_Compass'): [{'GEO': 220}], ('Iselda', 'Quill'): [{'GEO': 120}], ('Salubra', 'Lifeblood_Heart'): [{'GEO': 250}], ('Salubra', 'Longnail'): [{'GEO': 300}], ('Salubra', 'Steady_Body'): [{'GEO': 120}], ('Salubra', 'Shaman_Stone'): [{'GEO': 220}], ('Salubra', 'Quick_Focus'): [{'GEO': 800}], ('Leg_Eater', 'Fragile_Heart'): [{'GEO': 350}], ('Leg_Eater', 'Fragile_Greed'): [{'GEO': 250}], ('Leg_Eater', 'Fragile_Strength'): [{'GEO': 600}], ('Seer', 'Dream_Gate'): [{'ESSENCE': 900}], ('Seer', 'Awoken_Dream_Nail'): [{'ESSENCE': 1800}], ('Grubfather', 'Grubsong'): [{'GRUBS': 10}], ('Grubfather', "Grubberfly's_Elegy"): [{'GRUBS': 46}], ('Seer', 'Mask_Shard'): [{'ESSENCE': '1500'}], ('Grubfather', 'Mask_Shard'): [{'GRUBS': 5}], ('Seer', 'Vessel_Fragment'): [{'ESSENCE': 700}], ('Salubra', 'Charm_Notch'): [{'CHARMS': 25, 'GEO': 1400}, {'CHARMS': 18, 'GEO': 900}, {'CHARMS': 10, 'GEO': 500}, {'CHARMS': 5, 'GEO': 120}], ('Salubra', "Salubra's_Blessing"): [{'CHARMS': 40, 'GEO': 800}], ('Seer', 'Pale_Ore'): [{'ESSENCE': 300}], ('Grubfather', 'Pale_Ore'): [{'GRUBS': 31}], ('Grubfather', 'Rancid_Egg'): [{'GRUBS': 16}], ('Seer', 'Hallownest_Seal'): [{'ESSENCE': 100}], ('Grubfather', "King's_Idol"): [{'GRUBS': 38}], ('Seer', 'Arcane_Egg'): [{'ESSENCE': 1200}]}
```

Comment:
One of these costs is a string instead of an int, I had a generation failure during beatability check due to this

`('Seer', 'Mask_Shard'): [{'ESSENCE': '1500'}]`